modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -78,16 +122,42 @@ export default function ViewObjectDefinitions({
 		);
 	}
 
+	const getApiURL = () => {
+		let url: string = '';
+		if (selectedFolder.externalReferenceCode) {
```

Comment:
@rebsilva Please, add a blank line between the declaration of url and its usage/the block of its usage.